### PR TITLE
Missing T on the 3rd paragraph

### PR DESCRIPTION
**Pull request by silvermarkg**

### CODE REVIEW

File: Office365-Admin/dns/create-dns-records-using-windows-based-dns.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you host your own DNS records using Windows-based DNS, follow the steps in th
   
 To get started, you need to [find your DNS records in Windows-based DNS](#find-your-dns-records-in-windows-based-dns) so you can update them. Also, if you're planning to synchronize your on-premises Active Directory with Office 365, see [Non-routable email address used as a UPN in your on-prem Active Directory](#non-routable-email-address-used-as-a-upn-in-your-on-prem-active-directory).
   
-ouble with mail flow or other issues after adding DNS records, see [Troubleshoot issues after changing your domain name or DNS records](../get-help-with-domains/find-and-fix-issues.md). 
+Touble with mail flow or other issues after adding DNS records, see [Troubleshoot issues after changing your domain name or DNS records](../get-help-with-domains/find-and-fix-issues.md).
```
Comment (mypil) [inline Office365-Admin/dns/create-dns-records-using-windows-based-dns.md]:
```suggestion
Trouble with mail flow or other issues after adding DNS records, see [Troubleshoot issues after changing your domain name or DNS records](../get-help-with-domains/find-and-fix-issues.md). 
```

Comment (silvermarkg) [reply Office365-Admin/dns/create-dns-records-using-windows-based-dns.md]: Spelling was never my strong suit 😀

Comment (mypil) [reply Office365-Admin/dns/create-dns-records-using-windows-based-dns.md]: Can you please merge the commit or make the necessary changes on the PR following the suggestion made?  

Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT - There is a missing "r" in the word. The suggested changes to add this was not merged or added so the word "Trouble" is spelled as Toubled in the doc. I created PR #947 to add the missing. 

Thank you.

cc: @kenwith

**Pull request merged**